source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/RendererUtil.java

```diff
@@ -156,4 +157,17 @@ public void handleTag(boolean opening, String tag, Editable output, XMLReader xm
             }
         }
     }
+
+    static HorizontalAlignment computeHorizontalAlignment(RenderArgs renderArgs, HorizontalAlignment declaredAlignment)
```

Comment:
getHorizontalAlignment

Reply:
Same comment as above. get/set should be reserved for "property"-like accessors/setters